app/templates/components/chart-toggle.html

```diff
@@ -0,0 +1,18 @@
+<p class="text-corporate-blue text-center text-h2">
+  {{ toggleText }}
+  <select
+    id="{{ section_id }}_toggle"
+    class="bg-white text-center font-bold border-b-2 border-corporate-blue inline"
+    aria-controls="{{ section_id }}_ber {{ section_id }}_ger"
+    aria-label="Schalte um zwischen Berlin und Deutschland"
+  >
+    <option value="{{ section_id }}_ber">Berlin</option>
+    <option value="{{ section_id }}_ger">Deutschland</option>
+  </select>
+</p>
+<div id="{{ section_id }}_ber" class="">
+  {{ request.app.extra[chart_ids[0]]|safe }}
+</div>
+<div id="{{ section_id }}_ger" class="hidden">
+  {{ request.app.extra[chart_ids[1]]|safe }}
+</div>
```

Comment:
If this is hidden with CSS alone, we might want to add [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) as well, to signalize that this element is not visible at the moment.

Reply:
Actually it's `display:none`. It's one of tailwinds false friends. `visibility: hidden` would be` "invisible"`